README.md

```diff
@@ -1,20 +1,21 @@
 Validate an LSST DM processCcd.py output repository
 against a set of LSST Science Requirements Document Key Performance Metrics.
+and expected performance following the LSST Overview paper.
 
 ```
 setup validate_drp
 validateDrp.py CFHT/output
 ```
 
-will produces output, plots, JSON files analyzing the processed data in `CFHT/output`.  Metrics will be separately calculated for each filter represented in the repository.  Replace `CFHT/output` with your favorite processed data repository and you will get reasonable output.
+will produces output, plots, JSON files analyzing the processed data in `CFHT/output`.  Metrics will be separately calculated for each filter represented in the repository.  
+
+Replace `CFHT/output` with your favorite processed data repository and you will get reasonable output.
 
 One can run `validateDrp.py` in any of the following modes:
 1. use no configuration file (as above)
```

Comment:
If you want this to render as a list in Markdown you can add a blank line before the first list item.